website/api/views.py

```diff
@@ -474,40 +474,61 @@ class BugHuntApiViewset(APIView):
 
     def get_active_hunts(self, request, *args, **kwargs):
         hunts = (
-            Hunt.objects.values("id", "name", "url", "prize", "logo", "starts_on", "end_on")
+            Hunt.objects.values(
+                "id", "name", "url", "prize", "logo", "banner", "description", "starts_on", "end_on"
```

Comment:
It looks like `"id", "name", "url", "prize", "logo", "banner", "description", "starts_on", "end_on"` is the same for all methods. Could you extract it to a separate variable so that next time we would need to modify it just once?

Thank you!

Reply:
I have made the changes and extracted them to a list of string fields. I have tested it too and it was working, any other suggestions? 
 Thank you.